AUTH-README.md

```diff
@@ -18,6 +18,8 @@ Note that you must use the release of the Jetty-ALPN jar specific to the version
 
 An option is provided to use GRPC over plaintext without TLS. This is convenient for testing environments, however users must be aware of the secuirty risks of doing so for real production systems.
 
+ALPN is supported starting Android 4.4, but it only works well in 5.0 and higher. 
```

Comment:
Is it because OpenSSL is up-to-date, but the SSLSocketFactory has the ALPN bug, and the socket factory isn't part of Play Services Dynamic Security Provider?

If so, we should say something to that effect, or provide references.